generators/ci-cd/generator.ts

```diff
@@ -48,6 +45,20 @@ export default class CiCdGenerator extends BaseApplicationGenerator {
       sayHello() {
         this.log.log(chalk.white('🚀 Welcome to the JHipster CI/CD Sub-Generator 🚀'));
       },
+      validateSupportedCICD() {
+        if (this.jhipsterConfig.ciCd?.length > 0) {
+          if (
+            intersection(
+              command.configs.ciCd.choices.map(entry => entry.value),
+              this.jhipsterConfig.ciCd,
+            ).length !== this.jhipsterConfig.ciCd.length
+          ) {
+            throw new Error(
+              `error: command-argument value '${this.jhipsterConfig.ciCd}' is invalid for argument 'ciCd'. Allowed choices are github, jenkins, gitlab, azure, travis, circle.`,
```

Comment:
I don't know if "command-argument value" would make sense to me in an error message.

Reply:
Was an error thrown in previous version, I kept it like this